packages/components/src/test/properties/UniquePropertyValuesSelector.test.tsx

```diff
@@ -165,6 +165,21 @@ describe("UniquePropertyValuesSelector", () => {
     expect(queryByText("unique-values-property-editor.select-values")).to.not.be.null;
   });
 
+  it("sets empty value text if provided value is an empty string", async () => {
```

Comment:
doesn't `loads row with empty string as displayValue and sets it to an 'Empty Value' string` cover this?

Reply:
This test covers the creation of tag-label elements for empty display values, whereas `loads row with empty string as displayValue and sets it to an 'Empty Value' string` covers the empty string value that is displayed in the selector dropdown.